alibi_detect/utils/saving.py

```diff
@@ -89,7 +89,7 @@ def save_detector(detector: Data, filepath: Union[str, os.PathLike]) -> None:
     filepath
         Save directory.
     """
-    if 'backend' in list(detector.meta.keys()) and detector.meta['backend'] == 'pytorch':
+    if 'backend' in list(detector.meta.keys()) and detector.meta['backend'] in ['pytorch', 'sklearn']:
         raise NotImplementedError('Detectors with PyTorch backend are not yet supported.')
```

Comment:
It would be easy to do, as sklearn models are usually dill friendly. But as it currently stands the config PR will (temporarily) strip any saving support out again. I could also add save/load support for `sklearn` into the config PR, but it would increase the size of the PR...

Reply:
Ok let's do it later then.